src/engine/render.tsx

```diff
@@ -40,14 +40,18 @@ export interface IsomerBaseSchema {
   components: IsomerComponent[]
   sitemap?: Sitemap
   indexable?: string[] // specifies which keys are indexable
+  LinkComponent?: any // Next.js Link
```

Comment:
is there a reason why we don't want to type the `LinkComponent`?
like `<a>` or `NextLink`
it would allow us to have better typed props on the components like in `Navbar`

Reply:
Hmm yea, the only issue was to type as NextLink I need the dependency of nextjs, but this was supposed to be framework agnostic so I left it as any for now. Need a better way to type this